### PR TITLE
Smoke Tests

### PR DESCRIPTION
**Pull request by boopathi**
Duplicate of #352

### CODE REVIEW

**Comment by hzoo** [comment]
Hey @boopathi! It looks like one or more of your builds have failed. I've copied the relevant info below to save you some time.

* [Node vnode - linux](https://travis-ci.org/babel/babili/jobs/219055247)

**Comment by hzoo** [comment]
Hey @vigneshshanmugam! It looks like one or more of your builds have failed. I've copied the relevant info below to save you some time.

* [Node vnode - linux](https://travis-ci.org/babel/babili/jobs/219220416)

**Comment by hzoo** [comment]
Hey @boopathi! It looks like one or more of your builds have failed. I've copied the relevant info below to save you some time.

* [Node vnode - linux](https://travis-ci.org/babel/babili/jobs/219264179)
* [Node vnode - linux](https://travis-ci.org/babel/babili/jobs/219264180)
* [Node vnode - linux](https://travis-ci.org/babel/babili/jobs/219264181)

File: smoke/run.js [hunk 1]
```diff
@@ -21,6 +21,13 @@ const TESTS = [
         typeConstructors: false
       }
     }
+  },
+  {
+    dir: "jquery",
+    files: "dist/jquery.js",
+    build: "npm run build",
```
Comment (boopathi) [inline smoke/run.js]: there is no `npm install` step ? Probably we are missing that.

**Comment by hzoo** [comment]
Hey @boopathi! It looks like one or more of your builds have failed. I've copied the relevant info below to save you some time.

* [Node vnode - linux](https://travis-ci.org/babel/babili/jobs/219446975)

**Pull request merged**